dll/cpl/openglcfg/general.c

```diff
@@ -170,7 +170,8 @@ INT_PTR CALLBACK GeneralPageProc(HWND hWndDlg, UINT uMsg, WPARAM wParam, LPARAM
             return TRUE;
 
         case WM_COMMAND:
-            if (LOWORD(wParam) == IDC_RENDERER || IDC_DEBUG_OUTPUT)
+            if (LOWORD(wParam) == IDC_RENDERER ||
+                LOWORD(wParam) == IDC_DEBUG_OUTPUT)
                 PropSheet_Changed(GetParent(hWndDlg), hWndDlg);
```

Comment:
Put curly brackets around PropSheet_Changed() call?

Reply:
Agreed, though not this file style. Done.